nodejs/week1/avg.js

```diff
@@ -0,0 +1,20 @@
+const args = process.argv.slice(2);
+
+function findAvg(args){
+    const length = args.length ;
+    const argumentArray = args.map(arg => parseFloat(arg));
```

Comment:
instead of naming your variables after the abstract type. you can name them after their primitive types.
e.g;
`args` -> `numbersAsStr` or `numbersUnparsed`
`argumentArray` -> `numbersAsFloat` or `numbersParsed`
etc.

your code will become more readable, as the variables implicitly comment your code.